automations/js/package.json

```diff
@@ -3,5 +3,8 @@
     "@octokit/rest": "^18.9.0",
     "js-yaml": "^4.1.0",
     "node-fetch": "^2.6.7"
+  },
+  "volta": {
+    "node": "14.19.1"
   }
```

Comment:
Thanks for adding this, but should we use the same version as on the frontend (v16)?

Reply:
I thought so too initially but I think for now it's better to match the version of the actions themselves, which is Node 14. We could update those to 16 potentially, if GitHub supports it!